extension_templates/annotation.py

```diff
@@ -179,3 +182,28 @@ def _update(self, X, Y=None):
 
         # implement here
         # IMPORTANT: avoid side effects to X, fh
+
+    # todo: set default parameters, so that a test instance can be created
+    @classmethod
+    def get_test_params(cls):
+        """Get default parameters of the estimator.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Default parameters related to the estimator class
+        """
+        params = super().get_test_params()
+
+        # todo: set the default parameters for the estimators
```

Comment:
it should be better explained how `create_test_instance` uses the parameters, and what form the dictionary needs to take (needs to result in a constructed instance when called `MyClass(**params)`)

Reply:
@fkiraly Could you please review it now? I have only updated for annotation.py. If it is fine then I'll update for remaining also.

Reply:
ok - I think you want to write more precisely what the return is - "default parameters related to the estimator class" is too vague - related how? Suggestion, write sth close to what I wrote, i.e., that `MyClass(**params)` gets a test case, where `params` is...

Reply:
also, `create_test_instance` should be mentioned

Reply:
here's how I would write the docstring:

```
        """Return testing parameter settings for the estimator.

        Returns
        -------
        params : dict or list of dict, default = {}
            Testing parameters related to the estimator class.
            Each dict are parameters to construct an "interesting" test instance,
            that is, `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
            `create_test_instance` uses the first (or only) dictionary in `params`
        """
```

Reply:
I have updated the docstring to as you have written in annotation.py and _base.py.